lib/active_project/webhook_event.rb

```diff
@@ -4,17 +4,28 @@ module ActiveProject
   # Represents a standardized event parsed from a webhook payload.
   # Using Struct for simplicity for now. Could be a full class inheriting BaseResource if needed.
   WebhookEvent = Struct.new(
```

Comment:
We will rebuild the webhook in a future version, i tested it only in basecamp and jira. 

You can remove the Webhook in GH to have the test pass.

Reply:
I got the webhooks working I'm pretty sure, I'll double check.